llpc/util/llpcElfWriter.cpp

```diff
@@ -1118,6 +1193,142 @@ void ElfWriter<Elf>::Reinitialize()
     m_map[NoteName] = m_noteSecIdx;
 }
 
+uint32_t GetResourceOffset(uint32_t descSet, uint32_t binding, ResourceMappingNodeType type, const ResourceMappingNode* userResources, uint32_t
+    resourceCount);
```

Comment:
Should be static. Is there a strong reason for making this a forward declaration as opposed to just having the function definition here?

Reply:
Done.